### DISCUSSION TITLE
WebUI won't load when using a different port for docker

### DISCUSSION THREAD

**Discussion by vypergts** [tphakala/birdnet-go#272] [Q&A] [answered]
Hi, I was able to get this setup in docker on my Unraid server. However, I already have 8080 in use for something else and whenever I change it, the WebUI won't load at all. I saw there was a closed issue for this (#169) but it only pertained to the settings file? I am changing the port both on my docker template and in my config.yaml file. Is there someplace else it needs to be changed?

**Comment by tphakala** [comment]
There is no need to change port within birdnet-go settings when using Docker containers, just change externally exposed port on Docker start settings. Following command maps birdnet-go port 8080 to 9090.

docker run -ti \
  -p 9090:8080 \
  --env ALSA_CARD=0 \
  --env TZ="Europe/Stockholm" \
  --device /dev/snd \
  -v $HOME/BirdNET-Go-Volumes/config:/config \
  -v $HOME/BirdNET-Go-Volumes/data:/data \
  ghcr.io/tphakala/birdnet-go:latest

**Comment by vypergts** [comment]
Yes, that's normally what I have in the Unraid template (It's not exactly apples to apples with docker run but pretty close):
![image](https://github.com/user-attachments/assets/0eee266d-1d88-435a-9748-a224fffa8d8f)
It's running on that port:
![image](https://github.com/user-attachments/assets/084bb51f-2df0-4ccb-a00b-bbcb7b6be251)
But the WebUI isn't accessible:
![image](https://github.com/user-attachments/assets/f8d8d1a2-87ad-41a4-b30d-020aae6adccb)

**Comment by tphakala** [comment, marked as answer]
You have set BirdNET-Go HTTP port as 8778 but your unraid expects it to be 8080 (container port setting). Change BirdNET-Go port (in config.yaml) to 8080 and it should start working.

**Reply by vypergts** [reply]
Doh! You're right. That fixed it, thank you so much!

**Comment by normal-dot-brad** [comment]
@vypergts Do you mind giving a brief overview of how you installed it on Unraid ?
